src/app/domains/wallets/domains.wallets.gateway.ts

```diff
@@ -26,4 +27,20 @@ export class DomainsWalletsGateway {
      },
    ])
   }
+
+  public addNewWallet(name: string): Observable<IWalletApiResponse> {
```

Comment:
Here instead of passing `name` you should pass object. `name` is only one property, Wallet will have more properties soon so we will have to change a lot of places where only one string is required.
You can create new interface called `IWalletPayload`:
```typescript
export interface IWalletPayload {
    name: string;
}
```
Please do the same for `updateNewWallet` method. So there you will have such parameters: `updateNewWallet(id: number, payload: IWalletPayload)`.